composer.json

```diff
@@ -60,7 +60,12 @@
         "preferred-install": {
             "*": "dist"
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "symfony/flex": true,
```

Comment:
I know that this can sound a bit obsessive-compulsive, but can we get this list sorted plz?